packages/flutter/lib/src/material/about.dart

```diff
@@ -1230,9 +1230,9 @@ class _MasterDetailFlowState extends State<_MasterDetailFlow> implements _PageOp
   }
 
   MaterialPageRoute<void> _detailPageRoute(Object? arguments) {
-    return MaterialPageRoute<dynamic>(builder: (BuildContext context) {
-      return PopScope(
-        onPopInvoked: (bool didPop) {
+    return MaterialPageRoute<void>(builder: (BuildContext context) {
+      return PopScopeWithResult<void>(
+        onPopInvoked: (bool didPop, void result) {
```

Comment:
Having a void variable feels a bit weird when it happens due to a void API return. 

```suggestion
        onPopInvoked: (bool didPop, _) {
```

Reply:
That's true but we also have [avoid anonymous parameter names](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-anonymous-parameter-names) in the styleguide. I agree that `void result` is weird, though.